CONTRIBUTING.md

```diff
@@ -1,42 +1,67 @@
-# Contributing
+# Contributing to HedgeDoc
 
-When contributing to this repository, please first discuss the change you wish to make via issue,
-email, or any other method with the owners of this repository before making a change.
+HedgeDoc is a volunteer effort.
+We encourage you to pitch in and to help us making this project even better!
 
-Please note we have a [code of conduct](CODE-OF-CONDUCT.md), please follow it in all your
+Please note we have a [code of conduct][code-of-conduct], please respect it in all your
 interactions with the project.
 
-## Pull Request Process
+## Ways of contributing
+### Do you have questions about the project?
 
-1. Ensure you signed all your commits with Developer Certificate of Origin (DCO).
+* Feel free to post your question on our [community forum](community-forum) or join our [matrix community chat][matrix-support].
 
-2. Ensure any install or build dependencies are removed before the end of the layer when doing a
-   build.
+### Did you find a bug?
 
-3. Update the README.md with details of changes to the interface, this includes new environment
-   variables, exposed ports, useful file locations and container parameters.
+* **Ensure the bug wasn't already reported** by searching on GitHub under [Issues](issues).
 
-4. Increase the version numbers in any examples files and the README.md to the new version that this
-   Pull Request would represent. The versioning scheme we use is [SemVer](http://semver.org/).
+* If you're unable to find an open issue addressing the problem, [open a new one](new_issue). Be sure to use one of the templates we provide if your request applies to them.
 
-5. You may merge the Pull Request in once you have the sign-off of two other developers, or if you
-   do not have permission to do that, you may request the second reviewer to merge it for you.
+### Did you write a patch that fixes a bug?
 
-## Sign your work
+* Open a new GitHub pull request with the patch. See the section [submitting a pull request](#submitting-a-pull-request) for details on this.
+
+* Ensure the PR description is precise about the problem and your solution. Just fill out our template. That should cover the most important information.
+
+### Do you intend to add a new feature or change an existing one?
+
+* Suggest your idea via a new GitHub issue. After a confirmation about your idea, you can start writing code. Our maintainers and other project developers can provide useful details about the architecture and show you relevant issues and discussions.
+
+### Do you want to work on translations?
+
+* If you want to improve a translation or add a new translation altogether, we handle those via [POEditor](poeditor).
```

Comment:
This anchor need different brackets to work.

Reply:
Fixed here and at all other places!